.unacceptablelanguageignore

```diff
@@ -0,0 +1 @@
+NOTICE.txt
```

Comment:
If this is flagged due to the `master` we can just switch that to a commit sha instead